docs/source/tracking.rst

```diff
@@ -251,26 +251,37 @@ You run an MLflow tracking server using ``mlflow server``.  An example configura
 Storage
 -------
 
-An MLflow tracking server has two properties related to how data is stored: a backend store and
-an artifact store.
-
-The *backend store* (exposed as ``--backend-store-uri``) is where the *server* stores run and
-experiment metadata. For backward compatibility, ``--file-store`` option is an alias to this
-option. This can be a local path *file store* specified as ``./path_to_store`` or
-``file://path_to_store``, or a SQL connection string for a *database backed store*. For the
-latter, the argument must be a SQL connection string specified as
-``db_type://<user_name>:<password>@<host>:<port>/<database_name>``. Supported database types are
-``mysql``, ``mssql``, ``sqlite``, and ``postgresql``.
+An MLflow tracking server has two components for storage: a **backend store** and an
+**artifact store**.
+
```

Comment:
@mparkhe  We recommend using italics (*) rather than bold (**) for defining terms.

Reply:
Gotcha. Fixing.